### PR TITLE
test: Refactor some tests for AWS invoke to use runServerless.

### PR DESCRIPTION
**Pull request by lewgordon**
I tried taking a first stab at this.  One of the tests I moved out the common section as I thought it would be easier to implement the test in that way.  I figured a smaller PR would potentially help me get a feel for what your thoughts are on how to implement these tests.

Addresses: #8662

### CODE REVIEW

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Hello @lewgordon :wave: You're totally on the right path when it comes to implementing these :raised_hands: I've posted a few suggestions, but overall it looks good :raised_hands:

File: test/unit/lib/plugins/aws/invoke.test.js [hunk 1]
```diff
@@ -303,34 +245,78 @@ describe('AwsInvoke', () => {
...
       // https://github.com/serverless/serverless/blob/537fcac7597f0c6efbae7a5fc984270a78a2a53a/test/unit/lib/plugins/aws/invoke.test.js#L123-L129
     });
+  });
 
-    it('TODO: should log payload', async () => {
-      // Confirm by inspecting stdout, that it logs data as returned by our AWS SDK stub
-      // Replaces
-      // https://github.com/serverless/serverless/blob/537fcac7597f0c6efbae7a5fc984270a78a2a53a/test/unit/lib/plugins/aws/invoke.test.js#L236-L251
-      // https://github.com/serverless/serverless/blob/537fcac7597f0c6efbae7a5fc984270a78a2a53a/test/unit/lib/plugins/aws/invoke.test.js#L301-L315
+  it('should log payload', async () => {
```
[the remark this answers is not in the document]
Comment (lewgordon) [reply test/unit/lib/plugins/aws/invoke.test.js]: Sure, I think I follow you there. I kept the assertions too, though. Not sure if that now makes `should invoke AWS SDK with expected params` mostly redundant?

[the remark this answers is not in the document]
Comment (pgrzesik) [reply test/unit/lib/plugins/aws/invoke.test.js]: I agree that now we have redundancy/duplication across these two tests. I believe we could keep the `should invoke AWS SDK with expected params` as-is and remove assertions on a `stub` from `should log payload`, so it only confirms logs on stdout. What do you think?

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Looks great :raised_hands: Please see my last comment and I believe after that we'll be good to go :+1:

File: test/unit/lib/plugins/aws/invoke.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lewgordon) [reply test/unit/lib/plugins/aws/invoke.test.js]: I think that is reasonable!

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Looks great, thank you @lewgordon. Sorry for the trouble, but I think you'll need to rebase on top of master as we introduced `prettier` in `v2` yesterday and current linting/formatting validation is failing on your branch due to that.

**Review by pgrzesik — APPROVED** [review]
Thank you @lewgordon, awesome job :+1:

**Pull request merged**